packages/react/src/hooks/__tests__/useMediaQuery.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { cleanup } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useMediaQueryLessThan, useMediaQueryGreaterThan } from '../useMediaQuery';
+
+describe('useMediaQuery', () => {
+  beforeEach(cleanup);
+  it('useMediaQueryLessThan returns correct value', () => {
+    // By default the window.innerWidth is 1024px
```

Comment:
Can the tests control the window innerWidth value? Then it could be set before the tests and reset after.
I was just thinking of a scenario when the JSDom updates and the width becomes undefined or it changes

Reply:
You're right, it would be more safe. I saw people having problems with responsive testing so I just "went where the fence was at the lowest" 😄 I'll try this out.

Reply:
Updated, [check it out](https://github.com/City-of-Helsinki/helsinki-design-system/pull/960/commits/ed60051466e9f36c1ecd88dea89280747ad8e449).

Reply:
I'm not sure if the cleanup resets the global object, the source code defines it as: 
`Unmounts React trees that were mounted with render.`
I also get act-warnings when I run the tests. I'm not sure if it is this test or some other test that causes those 🤔 
`console.error
    Warning: An update to TestComponent inside a test was not wrapped in act(...)`

Reply:
Gah, you're right. Thanks for keeping me in check! Now these both are [fixed.](https://github.com/City-of-Helsinki/helsinki-design-system/pull/960/commits/1bf41b423e34bdf3f6483d328b72871bcaf941d4)